docs/usage/configuration/config_documentation.md

```diff
@@ -3815,6 +3815,8 @@ Sub-options for this setting include:
 * `system_mxid_display_name`: set the display name of the "notices" user
 * `system_mxid_avatar_url`: set the avatar for the "notices" user
 * `room_name`: set the room name of the server notices room
+* `room_avatar_url`: set the room avatar URL of the server notices room
+* `room_topic`: set the room topic of the server notices room
```

Comment:
Please document when these were added.

We should probably define `room_avatar_url` as an MXC URL?